content/docs/rfcs/14/README.md

```diff
@@ -38,6 +38,10 @@ The `timestamp` field MAY be filled out to signify the time at which the message
 This field holds the Unix epoch time in nanoseconds. 
 Omitting it means the timestamp is unspecified.
 
+The `ephemeral` field MAY be filled out to signify the transient nature of the message.
+Omitting it means the message is permananent, and must be added to the store.
```

Comment:
If the `false` value is to be explained here, then would be good to also explain what is the interpretation of the `true` value in the context of the store protocol. Borrowing the description from above:
> The message MAY be ignored by the store service node"

Reply:
Alternatively, the reader can be referred to the store protocol specs for how this flag is rendered.

Reply:
Addressed in [eeed3e2](https://github.com/vacp2p/rfc/pull/532/commits/eeed3e28fef460452ea7a4c58084d6ad7cf3f2e7)